pandas/core/indexes/timedeltas.py

```diff
@@ -409,7 +412,7 @@ def _add_datelike(self, other):
             result = checked_add_with_arr(i8, other.value,
                                           arr_mask=self._isnan)
             result = self._maybe_mask_results(result, fill_value=iNaT)
-        return DatetimeIndex(result, name=self.name, copy=False)
```

Comment:
what changed here?

Reply:
It just got indented (next line in the diff) b/c this branch is never reached _except_ for in the indented case.